lib/solidus_retail/order/generate_pos_order.rb

```diff
@@ -45,17 +45,37 @@ def create_order
 
```

Comment:
Why would this need to be under the app directory? This doesn't really rely on Rails at all